UI/cypress/e2e/mobile.js

```diff
@@ -0,0 +1,39 @@
+import { Given, When, Then, Step } from "@badeball/cypress-cucumber-preprocessor";
+
+Then("a user is using a {string}", (device) => {
+    cy.viewport(428, 926);
+});
+
+When("the search result page is displayed", () => {
```

Comment:
There is already a common `a user has arrived on the 'Search results' page` and this can be removed if the scenarios above are changed in line with review comment

Reply:
Removed